lib/src/models/exclude_rule.dart

```diff
@@ -1,22 +1,23 @@
-/// Model class for AvoidReturningWidgetsExclude parameters
-class AvoidReturningWidgetsExclude {
+
+/// Model class for ExcludeRule parameters
+class ExcludeRule {
```

Comment:
Let's think about better naming here, this isn't a rule.

`ExcludedIdentifiersParameter` maybe?

```suggestion
class ExcludeRule {
```